deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
Ah! Gotcha. I see where were missing each other. None of the host mounted volumes are the issue. The problem is the postgreql data dir, which is _not_ a host mounted volume. I _think_ we could convert it to a host mounted volume by setting `PGDATA` and mounting the dir we specify with `docker -v` as described here: https://hub.docker.com/_/postgres/.

In AWS, as long as we mount to the EBS volume, any transient data will disappear when the EBS volume is destroyed on instance termination.